permissions/plugin-testproject.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "testproject"
+github: "jenkinsci/testproject-plugin"
+paths:
+- "org/jenkins-ci/plugins/testproject"
```

Comment:
https://github.com/jenkinsci/testproject-plugin/blob/master/pom.xml#L10-L11

Reply:
@batmat, fixed.

Reply:
Note that in this case, the more correct fix would have simply been to remove the `groupId` override. But it gives the same. So LGTM